app/main.py

```diff
@@ -1,8 +1,32 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        Person(person["name"], person["age"])
+        print(person)
+
+    for person in people:
+        instance = Person.people[person["name"]]
+
+        if "wife" in person.keys() and person["wife"] is not None:
```

Comment:
```suggestion
        if "wife" in person and person["wife"] is not None:
```
`.keys()` by default